inference-engine/src/gna_plugin/gna_graph_compiler.cpp

```diff
@@ -239,9 +239,13 @@ void GNAGraphCompiler::ConvolutionPrimitive(InferenceEngine::CNNLayerPtr layer)
         inputs->getLayout() != Layout::NC) {
         THROW_GNA_LAYER_EXCEPTION(layer) << "with layout " << inputs->getLayout() << " isn't currently supported on GNA";
     }
-    if (inputs->getLayout() != outputs->getLayout()) {
+
+    /*if (outputs->getLayout() != InferenceEngine::Layout::NHWC) {
+        THROW_GNA_LAYER_EXCEPTION(layer) << "GNA output is transposed to NHWC but output layout is " << outputs->getLayout();
+    }*/
+    /*if (inputs->getLayout() != outputs->getLayout()) {
         THROW_GNA_LAYER_EXCEPTION(layer) << "I/O layout mismatch: " << inputs->getLayout() << " vs " << outputs->getLayout();
-    }
+    }*/
```

Comment:
make sure that no code is left commented if it is not needed remove it, otherwise uncomment if this is just a temporary then ignore this.